template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApp-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json.schemastore.org/dotnetcli.host",
+    "symbolInfo": {
+        "target-framework-override": {
+            "isHidden": "true",
+            "shortName": ""
+        },
+        "no-restore": {
+            "shortName": ""
+        },
+        "langVersion": {
+            "shortName": ""
+        },
+        "nullable": {
+            "shortName": ""
+        }
+    },
+    "usageExamples": [
+        "--framework net6.0"
```

Comment:
This example existed before, however it is not needed now, consider whether we should keep it or remove.